assets/js/components/BatchEdit/Remove.js

```diff
@@ -27,15 +27,15 @@ export default function BatchEditRemove({
       </UIFormField>
 
       {removeItems.length > 0 && (
-        <div className="content">
-          <ul>
+        <div className="content has-background-danger-light">
+          <ul className="py-4">
             {removeItems.map((item) => {
               const { label, role, term } = splitFacetKey(item);
 
               return (
-                <li key={item} className="has-text-danger">
-                  {`${label} - ${term} - ${role}`}
-                </li>
+                <li key={item}>{`${label} - ${term} ${
+                  role ? "- " : ""
+                } ${role}`}</li>
```

Comment:
@katdivyareddy10 It's small, but can we tighten up this ternary?  To look something like:
```
<li key={item}>{`${label} - ${term}`} {role ? `- ${role}` : ""}</li>
```